apps/dashboard/src/modules/user/views/UserLandingView.vue

```diff
@@ -48,6 +51,14 @@ const getUserMutations = async (take: number, skip: number):
   return userStore.getCurrentUser.financialMutations;
 };
 
+const gewisUser: Ref<GewisUserResponse | undefined> = ref(undefined);
+onMounted(async () => {
+
+  await apiService.user.getIndividualUser(authStore.getUser?.id!).then((res) => {
```

Comment:
Next time: just put the ! after the getUser instead of first the ? and then either way the ! after id

Reply:
Alright, you want me to change it?